core/src/components/badge/badge.common.scss

```diff
@@ -1,3 +1,4 @@
+@use "../../themes/ionic/ionic.globals.scss" as globals;
```

Comment:
This file is for all themes so we do not use globals here since globals is for ionic only.

```suggestion
```